Resources/docs/extend_webpack_encore.md

```diff
@@ -0,0 +1,81 @@
+## Extend Webpack Encore
+
+If you are going to use your customized webpack-encore configuration and
+want to take advantage of all the libraries imported from AdminLTEBundle, 
+you can easily extend its configuration.
+
+### Create your webpack.config.js
+
+First of all, create your own webpack.config.js, as in [Symfony documentation](http://symfony.com/doc/current/frontend/encore/simple-example.html):
+
+```js
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+    .setOutputPath('public/builds/')
+    .setPublicPath('/builds')
+
+    // this will be your app!
+    .addEntry('app', './assets/js/app.js')
+    .autoProvidejQuery()
+    .enableSourceMaps(!Encore.isProduction())
+    .cleanupOutputBeforeBuild()
+    .enableBuildNotifications()
+    // You need sass loader!
+    .enableSassLoader()
+;
+
+module.exports = Encore.getWebpackConfig();
+```
+
+Now you need to create (or update) your `package.json`. If you don't have one, copy
+[package.json](../../package.json) from AdminLTEBundle. If you already have one, then
+integrate it with the packages listed in `vendor/kevinpapst/adminlte-bundle/package.json`.  
+
+Then create your main app and require adminlte:
+```js
+// assets/js/app.js
+require('../../vendor/kevinpapst/adminlte-bundle/Resources/assets/admin-lte');
+
+```
+
+Then, if you didn't already did it, install all packages and build your assets:
+
+```bash
+user@devpc:/var/www/symfony-project$ yarn install
+[...]
+user@devpc:/var/www/symfony-project$ ./node_modules/.bin/encore production
+
+```
+### Correct the assets path
+
+Now you have to update your assets path. To do this, create a new template
+that is going to extend AdminLTEBundle main template:
+
+```yaml
```

Comment:
remove `yaml` as syntax or replace with `twig` (as done in the next code example) if that is a supported language